tools/osv-scanner/cmd/osv-scanner/config.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"log"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+)
+
+type Config struct {
+	IgnoredVulnIds []string
+}
+
+// TryLoadConfig tries to load config in `target` or any of it's parent dirs
+// `target` will be the key for the entry in configMap
+// Will shortcut and return "" if globalConfig is not nil
+func TryLoadConfig(target string, configMap map[string]Config) string {
```

Comment:
Looking at https://go.dev/doc/effective_go#errors (and go/go-style/best-practices#program-init internally), I think it's preferable to return an error or an ok boolean to main, rather than abending within this function.

Please adjust this to either return the configuration and an error or boolean indicating success.